### ISSUE TITLE
Sierra Leone client CANNOT use the APP and have developed disparate system.

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/MentorApp#511]
Hi - So I will post more info as I have it....

Today I spoke with Sierra Leone.
Since late Janaury they have not been successful in using our APP.

They open the app from the desktop icon - Click on it - It opens (seemed very fast) - They can see the dashboard animation.
They click on "Create life map" button
Choose the Sierre Leone Survey and the app crashes.

This is super important for us to resolve - The client has developed their own system.
Using an older downloaded version of the app they show the user the PS Indicator screens - Then alongside they use an ODK collect to Google Sheets.
We will need to rectify the bug and recoup the data.

IMPORTANT I have tested the app for Sierra Leone in DEMO and DO NOT SEE AN ISSUE.
Could it be something related to the prod server?

I have requested:
CLIENT: Phone Information regarding the BUG
TEAM: Sentry Information from Dan (we saw the crash twice between 08:30 am colombia timeand 09:00AM Colombia time Tuesday 9 April)
CLIENT: Can they try with a different device to download
CLIENT AND FP TEAM : Log in Credentials so we can reproduce
CLIENT: Reporting info from the ODK Collect (so we can review thier needs in this respect once we have the bug solved!)

**Comment by stacylorraineprendeville** [comment]
@pA1nD @Danman7 anything else we need on this let me know....

**Comment by stacylorraineprendeville** [comment]
News from the client:

Thanks for speaking with us earlier. My personal mobile device is a Pixel 2 running Android version 9, Build number PQ2A.190405.003.

So actually, I have managed to get it to work! When I read your point on number 3, I decided to try logging out and back in again, and that seemed to solve it. I can now see that there are updates to the app! I will continue to test the app this week. We used the Survey 1 login. Apologies if the earlier bug was alarming for you -- I didn't know that I had to log out and log back in for the app to work.

The enumerators are currently in a different city and I am nervous about asking them to test on their tablet devices remotely while I am not there, as troubleshooting is very difficult over text or the phone. But I will try and see if the app will work on other Android devices this week.

So we need to resolve the issue with logging in and back out:
Is this:
https://app.zenhub.com/workspaces/-stoplight-global-5c11ac5d4b5806bc2bfd00bd/issues/fundacionparaguaya/mentorapp/252
